default_app/index.ts

```diff
@@ -1,30 +0,0 @@
-async function getOcticonSvg (name: string) {
```

Comment:
why move this to the preload script if the code can run just fine in the web context?

Reply:
I get that we want to use the new API, but that's only strictly necessary for getting the versions. On the other hand that logic is also default-app specific... OK, this at least moves the code into a single place :)